dm/worker/subtask.go

```diff
@@ -652,7 +652,7 @@ func (st *SubTask) Update(ctx context.Context, cfg *config.SubTaskConfig) error
 
 // OperateSchema operates schema for an upstream table.
 func (st *SubTask) OperateSchema(ctx context.Context, req *pb.OperateWorkerSchemaRequest) (schema string, err error) {
-	if st.Stage() != pb.Stage_Paused && req.Op != pb.SchemaOp_ListMigrateTargets {
+	if !(st.Stage() == pb.Stage_Paused || st.stage == pb.Stage_Running && req.Op == pb.SchemaOp_ListMigrateTargets) {
```

Comment:
new and use `ErrWorkerNotPausedAndRunningStage`?

Reply:
no need, because the task didn't start yet

Reply:
it's hard to remember the priority of different boolean operators, can you refine the check? like 

switch req.Op {
...
}